packages/synapse-react-client/src/style/components/synapse_form_wrapper/_synapse-form-wrapper.scss

```diff
@@ -255,8 +255,8 @@
 
     .help-block {
       font-style: italic;
-      font-weight: lighter;
-      font-size: 1.25rem;
+      font-weight: 400;
+      font-size: 14px;
```

Comment:
This text was extremely hard to read

Before:
![image](https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/17580037/a770e0f0-8b96-42db-83a6-016338d8d3a8)


After:

![image](https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/17580037/e1dff218-9830-42b2-9bad-9cc59caa2cba)